docs/source/helpers.rst

```diff
@@ -0,0 +1,14 @@
+=======
+Helpers
+=======
+
+Helpers are higher level functions built for reducing recurring code boilerplate.
+We add helpers as method of ``CSTNode`` or ``libcst.helpers`` package based on those principles:
+  - ``CSTNode`` method: simple, read-only and only require data of the direct children of a CSTNode.
+  - ``libcst.helpers``: node transforms or require recursively traversing the syntax tree.
+
+libcst.helpers
+--------------
+
+.. autofunction:: libcst.helpers.module.insert_header_comments
```

Comment:
We have this documented inside the codemods.rst file. If we are making a helpers document, we should remove it from there.